admin/system/edit-berita.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+    include("connection.php");
+    session_start();
+
+    $id_berita = $_GET["id"];
+    $judul_berita = $_POST["judul_berita"];
+    $kategori = $_POST["kategori"];
```

Comment:
Codacy found an issue: [Detected usage of a possibly undefined superglobal array index: $_POST["kategori"]. Use isset() or empty() to check the index exists before using it](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)